src/components/routeComponents/memberInformationGrid/columnConfig/ColumnConfiguration.jsx

```diff
@@ -218,7 +251,7 @@ class ColumnConfiguration extends Component {
 			return initialColumnConfigKeys.includes(key);
 		});
 
-		const chunkLength = Math.ceil(columnList.length / 4);
+		const chunkLength = Math.ceil(columnList.length / 1);
```

Comment:
No need to divide it by 1. Haha!
```suggestion
		const chunkLength = Math.ceil(columnList.length);
```

Reply:
Removed.